### ISSUE TITLE
Is compose-cupertino supported for web platform?

### ISSUE THREAD

**Issue by loongonthefly** [alexzhirkevich/compose-cupertino#69]
Thank you build this great project.
I want written a project using compose-cuperino.
Can it run on the web?

I want new project to iOS / Android and Web platform.

thanks again.

**Comment by schott12521** [comment]
Yes, this library supports KMP for web. The example app can be built and run for web.
